webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue

```diff
@@ -289,6 +289,7 @@ export default {
           return this.$nextTick();
         })
         .finally(() => {
+          document.title = this.$t('news.window.title', {0: this.news.title});
```

Comment:
Should you ensure that you display only the 100 first chars of the news title ?

Reply:
It display the same like the activity